src/components/ReportActionItemIOUPreview.js

```diff
@@ -94,61 +94,65 @@ const ReportActionItemIOUPreview = ({
 
     const managerName = lodashGet(
         personalDetails,
-        [iouReport.managerEmail, 'displayName'],
+        [iouReport.managerEmail, 'firstName'],
         iouReport.managerEmail ? Str.removeSMSDomain(iouReport.managerEmail) : '',
     );
     const ownerName = lodashGet(
         personalDetails,
-        [iouReport.ownerEmail, 'displayName'],
+        [iouReport.ownerEmail, 'firstName'],
```

Comment:
Ah, this doesn't cover the case where the user hasn't set a first/last name:

![Screenshot 2021-05-24 at 12 04 35](https://user-images.githubusercontent.com/10736861/119338938-818ce200-bc88-11eb-9b3f-7230889616f6.png)

Reply:
Ah oops. Nice catch!

Reply:
Updated! 
![image](https://user-images.githubusercontent.com/16747822/119382090-6fa24380-bc76-11eb-8d13-2acd22a6b8e3.png)